neqo-transport/src/qlog.rs

```diff
@@ -104,8 +104,8 @@ fn connection_started(qlog: &NeqoQlog, path: &PathRef) {
             protocol: Some("QUIC".into()),
             src_port: p.local_address().port().into(),
             dst_port: p.remote_address().port().into(),
-            src_cid: Some(format!("{}", p.local_cid())),
-            dst_cid: Some(format!("{}", p.remote_cid())),
+            src_cid: p.local_cid().map(|id| format!("{id}")),
+            dst_cid: p.remote_cid().map(|id| format!("{id}")),
```

Comment:
Nit, only related to this pull request:

`ConnectionId` implements `Display`. Instead of going through the full string interpolation macro `format!`, one might as well just call `ToString::to_string`:



```suggestion
            src_cid: p.local_cid().map(ToString::to_string),
            dst_cid: p.remote_cid().map(ToString::to_string),
```

See also https://doc.rust-lang.org/std/string/trait.ToString.html.

Intuitively, I would expect `ToString` to be faster than `format!`. Though that would need a benchmark proof.

Feel free to ignore.